packages/sandbox/buildHelpers/createRollupConfig.js

```diff
@@ -1,3 +1,5 @@
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
curious if you tested this PR out with the built npm packages? Would be worth doing before merge. 

You can grab the codesandbox urls for the npm packages from the github action that left a comment here. You would have to push a commit with the urls changed to your local embed running from `studio-ui` on your branch, grab the code sandbox urls, test out either in code sandbox or in a local webapp by replacing the ... in `@apollo/explorer: '...'` or sandbox with the urls you copied.

Reply:
When I use the URL from codesandbox, it works after I add these to my local web app's dependencies.
```
"subscriptions-transport-ws": "0.11.0",
"graphql-ws": "5.9.1",
```

When I use the file generated from `npm pack`, I get `Can't resolve '@apollo/sandbox/react' in '/Users/william/Apollo/test/embedded-explorer-test/src'`.

Reply:
Thanks! Yeah I'm still getting `Module not found: Error: Can't resolve '@apollo/sandbox/react' in '/Users/william/Apollo/test/embedded-explorer-test/src'` after `npm run build` then `npm pack`.

Is it normal for `graphql-ws` and `subscriptions-transport-ws` to be needed as dependencies? Should it be already bundled within @apollo/explorer and @apollo/sandbox?

Reply:
ah I think since you are packing within the `packages` folder and those don't have dependencies (we use[ npm workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces/) - and all the deps for both of those packages that are shared are in the root package.json. 

Hmm I am getting the same thing on main 🤔 if the code sandbox looks good I think were okay